cli/tests/environment-managed.bats

```diff
@@ -232,6 +210,22 @@ EOF
 @test "m8: search works in managed environment" {
   make_empty_remote_env
 
-  run "$FLOX_BIN" search hello;
-  assert_success;
+  run "$FLOX_BIN" search hello
+  assert_success
+}
+
+# ---------------------------------------------------------------------------- #
+
+# Make sure we haven't activate
+# bats test_tags=managed,activate,managed:activate
+@test "m9: activate works in managed environment" {
+  make_empty_remote_env
+  "$FLOX_BIN" install hello
+
+  # TODO: flox will set HOME if it doesn't match the home of the user with
+  # current euid. I'm not sure if we should change that, but for now just set
+  # USER to REAL_USER.
+  SHELL=bash USER="$REAL_USER" run -0 expect -d "$TESTS_DIR/activate/hello.exp" "$PROJECT_DIR";
+  assert_output --regexp "bin/hello"
```

Comment:
blocking: because of how we're wrapping Path with Managed with Remote environments, I think it would be good to test we put the symlink in the right place